cdc/api/v2/changefeed.go

```diff
@@ -895,6 +896,135 @@ func (h *OpenAPIV2) status(c *gin.Context) {
 	})
 }
 
+// transformer timestamp to readable format
+func transformerTime(timestamp int64) string {
```

Comment:
Perhaps oracle.GetTimeFromTS is a better choice?

Reply:
We need to extract the physical time and then transform it as a readable format, so I think oracle.GetTimeFromTS may be not enough for it.

Reply:
Oh, I get your point. Let me make a try